backend/test/app/services_teams_test.clj

```diff
@@ -159,4 +166,60 @@
 
 
 
+(t/deftest query-team-invitations
+  (let [prof  (th/create-profile* 1 {:is-active true})
+        team     (th/create-team* 1 {:profile-id (:id prof)})
+        data {::th/type :team-invitations
+              :profile-id (:id prof)
+              :team-id (:id team)}]
+
+    ;;insert an entry on the database with an enabled invitation
+    (db/exec-one! th/*pool*
+                  ["insert into team_invitation(team_id, email_to, role, valid_until) values (?, ?, ?, ?)"
+                   (:team-id data) "test1@mail.com" "editor", (dt/in-future "48h")])
+
+    ;;insert an entry on the database with an expired invitation
+    (db/exec-one! th/*pool*
+                  ["insert into team_invitation(team_id, email_to, role, valid_until) values (?, ?, ?, ?)"
```

Comment:
This simple instert can be done with `db/insert!`

```clojure
(db/insert th/*pool* :team-invitation
           {:team-id ...
            :email-to ...
            ...})
```

Reply:
fixed